### PR TITLE
📖 Clarify readymade amp-video-iframe integration APIs

### PR DESCRIPTION
**Pull request by alanorozco**
Video.js integration takes a `<video>` element, Jwplayer integration takes a `jwplayer` instance object.

### CODE REVIEW

File: extensions/amp-video-iframe/amp-video-iframe.md [hunk 1]
```diff
@@ -168,16 +168,24 @@ it lives on the non-AMP document that is iframed.
 If you're using a common video framework like JwPlayer or Video.js, you can
 simply call `listenTo` for a basic integration:
 
-##### For JwPlayer:
+##### For JwPlayer
+
+Pass in your [`jwplayer` instance object](https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/)
+through the signature `ampIntegration.listenTo('jwplayer', myJwplayer)`. The `ampIntegration` object then knows how
+to setup the player through the instance API.
```
Comment (wassgha) [inline extensions/amp-video-iframe/amp-video-iframe.md]: Worth throwing in a `{% call callout('Important', type='caution') %}` here maybe?

Comment (alanorozco) [reply extensions/amp-video-iframe/amp-video-iframe.md]: Caution about what? :P

Comment (alanorozco) [reply extensions/amp-video-iframe/amp-video-iframe.md]: (added a general `note` above.)

File: extensions/amp-video-iframe/amp-video-iframe.md [hunk 2]
```diff
@@ -166,18 +166,30 @@ it lives on the non-AMP document that is iframed.
 ### Simple integrations
 
 If you're using a common video framework like JwPlayer or Video.js, you can
-simply call `listenTo` for a basic integration:
+simply call `listenTo` for a basic integration.
```
Comment (CrystalOnScript) [inline extensions/amp-video-iframe/amp-video-iframe.md]: omit `simply`.

File: extensions/amp-video-iframe/amp-video-iframe.md [hunk 3]
```diff
@@ -166,18 +166,30 @@ it lives on the non-AMP document that is iframed.
 ### Simple integrations
 
 If you're using a common video framework like JwPlayer or Video.js, you can
```
Comment (CrystalOnScript) [inline extensions/amp-video-iframe/amp-video-iframe.md]: Mind adding a link the frameworks?

**Pull request merged**